src/test/regress/sql/seclabel_non_maindb.sql

```diff
@@ -0,0 +1,72 @@
+-- SECLABEL
+--
+-- Test suite for running SECURITY LABEL ON ROLE statements from non-main databases
+
+SET citus.enable_create_database_propagation to ON;
+
+CREATE DATABASE database1;
+CREATE DATABASE database2;
+
+\c - - - :worker_1_port
+SET citus.enable_create_database_propagation to ON;
+CREATE DATABASE database_w1;
+
+
```

Comment:
Given that SET commands only alters the value of the GUC for the current session (and \c means switching into a different connection/session), those two lines don't have any affect for the rest of the tests.

```suggestion
```